debs/generate_debian_package.sh

```diff
@@ -105,12 +107,12 @@ build() {
             FILE_PATH="${DEB_PPC64LE_BUILDER_DOCKERFILE}"
         else
             echo "Invalid architecture. Choose: x86_64 (amd64 is accepted too) or i386 or ppc64le."
-            clean 1
+            return 1
         fi
-        build_deb ${BUILD_NAME} ${FILE_PATH} || clean 1
+        build_deb ${BUILD_NAME} ${FILE_PATH} || IS_CORRECT=1
```

Comment:
This variable is unused. Also, this must be a `return 1`

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/329/commits/7ae0d158f1f8aa5b4c452b4ca85f9731cbced387.